apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -81,6 +81,8 @@ class SectionProgress extends Component {
 
     const levelDataInitialized = scriptData && studentLevelProgress;
 
+    const linkToOverview = scriptData ? scriptData.path : null;
```

Comment:
I did this because there's a small window of time when you first load the page when we're still fetching `scriptData`, `scriptData` is undefined, so `scriptData.path` can't throws an error.